samples/account-linking/csharp/Source/AccountLinking.Service/Services/TemporaryAuth/TempAuthToken.cs

```diff
@@ -0,0 +1,6 @@
+namespace Microsoft.Teams.Samples.AccountLinking.Service.TempAuth;
+
+public class TempAuthToken
```

Comment:
Can you add doc comments to explain the purpose? It'll be covered in the overview, but I think it's helpful to have pointers in code too for those starting by browsing code.